guacamole-ext/src/main/java/org/apache/guacamole/net/auth/Credentials.java

```diff
@@ -236,4 +236,35 @@ public void setRemoteHostname(String remoteHostname) {
         this.remoteHostname = remoteHostname;
     }
 
+    /**
+     * Returns whether this Credentials object does not contain any specific
+     * authentication parameters, including HTTP parameters and the HTTP header
+     * used for the authentication token. An authentication first, anonymous,
```

Comment:
Did some words get cut out of this when it was moved over? 

I'm not quite sure what
> An authentication first, anonymous, credential-less authentication attempt that results in the initial login screen rendering.

means here.

Reply:
> Did some words get cut out of this when it was moved over?

Yep - not sure how. Fixed via rebase.